ironfish-http-api-v2/tsconfig.json

```diff
@@ -1,10 +1,21 @@
 {
-  "extends": "../config/tsconfig.base.json",
   "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
     "experimentalDecorators": true,
+    "forceConsistentCasingInFileNames": true,
+    "incremental": true,
+    "lib": ["dom", "dom.iterable", "ES2020"],
+    "module": "commonjs",
+    "moduleResolution": "node",
     "outDir": "build",
     "resolveJsonModule": true,
     "rootDir": "src",
+    "skipLibCheck": true,
+    "sourceMap": true,
+    "strict": true,
+    "target": "ES2020",
     "tsBuildInfoFile": "./build/tsconfig.tsbuildinfo"
   },
   "include": ["src"],
```

Comment:
Why is this one getting materialized and options duplicated? Does it have issues with extending configs?

Reply:
I was thinking we should deploy only this package to Heroku since the entire repository has some dependencies that aren't necessary for the API (ex. wasm). It seems cleaner to have everything deployed directly from this folder (which is why I refactored `tsconfig.json`) rather than pushing a subtree. What do you think?